components/Navigation.js

```diff
@@ -0,0 +1,46 @@
+import styled from "styled-components";
+import NavigationIcons from "./Icons/NavigationIcons";
+import Link from "next/link";
+
+export default function Navigation() {
+  return (
+    <StyledNavigationContainer>
+      <StyledNavigationLink href="/">
+        <NavigationIcons variant={"home"} color={"#BABABA"} size={32} />
+      </StyledNavigationLink>
+
+      <StyledNavigationLink href="/recipe/create">
+        <NavigationIcons variant={"create"} color={"#BABABA"} size={32} />
+      </StyledNavigationLink>
+
+      {/* THIS LINE OF CODE IS COMMENTED OUT,IT WILL COME IN THE NEXT USER STORY WHEN WE DIRECT THE USER TO THE FAVORITES PAGE */}
+      {/* <Link href="/">
+        <NavigationIcons variant={"favorites"} color={"#BABABA"} size={32} />
+      </Link> */}
+    </StyledNavigationContainer>
+  );
+}
+
+const StyledNavigationContainer = styled.div`
```

Comment:
We might want to consider a `footer` html element here.

We could then nest a `nav` html element inside. 

And then i would consider wrapping your links in a `ul`. 

i.e. following this structure. 

``` html
<footer>
  <nav>
    <ul>
      <li><a href="#home">Home</a></li>
      <li><a href="#about">About</a></li>
      <li><a href="#contact">Contact</a></li>
    </ul>
  </nav>
</footer>
```